src/battle-tooltips.ts

```diff
@@ -1316,13 +1316,16 @@ class BattleTooltips {
 	 * Gets the proper current type for moves with a variable type.
 	 */
 	getMoveType(move: Move, value: ModifiableValue, forMaxMove?: boolean | Move): [TypeName, 'Physical' | 'Special' | 'Status'] {
+		const pokemon = value.pokemon!;
```

Comment:
```suggestion
		const pokemon = value.pokemon;
```

Pretty sure this works without the `!`

Reply:
yeah you're right. actually there's a check to see if the variable is null later so it's better to not have it.